configuration/cluster/gke/composition.yaml

```diff
@@ -40,9 +40,9 @@ spec:
         - fromFieldPath: spec.writeConnectionSecretToRef.namespace
           toFieldPath: spec.writeConnectionSecretToRef.namespace
         # Create this cluster in the Network and Subnetwork referenced by network ID
-        - fromFieldPath: "spec.parameters.networkRef.id"
+        - fromFieldPath: "spec.id"
```

Comment:
Did had a chance to check/figure out whether we could remove `id` field of top level XR spec in favor of `metadata.name` ?

Reply:
My understanding might be wrong, but I assumed it's required for Upbound Cloud UI. If you think it's not related, I'm happy to give it a try

Reply:
I don't think it is that important ATM but would be nice to track with a simple issue not to forget.